views/score.erb

```diff
@@ -0,0 +1,19 @@
+<a href="/">Home</a>
```

Comment:
In place of this here and in the score_multiple view, it would be great to use a navigation bar in your overall layout. [This book](https://www.railstutorial.org/book/filling_in_the_layout) has some good materials on where you can get started w/ this now using rrails.